lab_gurpreet/test/test.js

```diff
@@ -0,0 +1,14 @@
+const expect = require('chai').expect;
+const newCowsay = require('../lib/index');
+const cowsay = require('cowsay');
+describe('the greet function', function() {
+  describe('#parse()', function(){
```

Comment:
I'm not quite sure what the #parse() is supposed to mean? Generally, you just want to describe your test here.